### DISCUSSION TITLE
How will we get MV2 extensions after removal?

### DISCUSSION THREAD

**Discussion by RobertPPAP** [Alex313031/thorium#797] [Q&A] [answered]
You said on m126.0.6478.231 you'll bring back MV2 when it's removed, so here are some questions
How will we get popular extensions that only work with MV2 (mainly uBlock Origin)?
Will uBlock Origin still be preinstalled? I checked the code for preinstalling it and it downloads it from the Chrome Web Store.
Will it be possible for MV2 extensions to still auto update?

**Comment by Alex313031** [comment, marked as answer]
@Robert-CMDedu  I will be restoring the MV2 code after it is removed. However, MV2 extensions will have to be installed manually. So I will probably just offer a .crx for ubock origin MV2.

**Reply by mzso** [reply]
Unless you plan to back up all MV2 extensions people surely will surely have trouble finding their favorite MV2 extensions.
